docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -30,13 +30,29 @@ NOTE: PacketFence PKI is available on the PacketFence repository as the package
 Installing 
 ^^^^^^^^^^
 
-First of all, you will need to install the package, use the following command matching your OS.
+First of all you have the choice to install the packetfence-pki on a stand alone machine or on the PacketFence machine.
+
+For the stand alone you need to open the port 9393 in tcp to be able to reach the pki.
```

Comment:
Be just a bit more specific and say that it needs to be opened in the OS firewall (in our case iptables)